modular_skyrat/master_files/code/modules/client/preferences/headshot.dm

```diff
@@ -0,0 +1,41 @@
+/datum/preference/text/headshot
+	category = PREFERENCE_CATEGORY_NON_CONTEXTUAL
+	savefile_identifier = PREFERENCE_CHARACTER
+	savefile_key = "headshot"
+	/// Assoc list of ckeys and their link, used to cut down on chat spam
+	var/list/stored_link = list()
```

Comment:
Are you sure this works the way you think it does? Are you sure that preference is a singleton?

Reply:
Yep